miden-tx/src/errors/tx_kernel_errors.rs

```diff
@@ -136,11 +136,11 @@ pub const TX_KERNEL_ERRORS: [(u32, &str); 87] = [
 
     (ERR_FAUCET_BURN_CANNOT_EXCEED_EXISTING_TOTAL_SUPPLY, "Asset amount to burn can not exceed the existing total supply"),
     (ERR_FAUCET_BURN_NON_FUNGIBLE_ASSET_CAN_ONLY_BE_CALLED_ON_NON_FUNGIBLE_FAUCET, "The burn_non_fungible_asset procedure can only be called on a non-fungible faucet"),
-    (ERR_FAUCET_INVALID_STORAGE_OFFSET, "Storage offset is invalid for a faucet account (0 is prohibited as it is the reserved data slot for faucets)"),
+    (ERR_FAUCET_INVALID_STORAGE_OFFSET, "for faucets)"),
```

Comment:
Currently, only the line directly above error messages is parsed as the error, so we have two options:
- Allow error messages to go beyond 100 characters,
- or implement multi-line comment parsing in the build script. I briefly tried when I implemented it but wasn't able to come up with a regex that worked, but it surely is possible to do this.

Reply:
Ah, my bad, I forgot that we parse just one line for the error message. For now I'll roll back these changes, but probably it makes sense to create a new issue for this.